tests/collections/nlp/test_huggingface.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from unittest import TestCase
+
+import pytest
+
+import nemo.collections.nlp as nemo_nlp
+
+
+class TestHuggingFace(TestCase):
```

Comment:
Should we have tests for example and models or everything?

Reply:
I covered just a bare minimum, let's sync offline for what's missing